2016/ideas-list-swc.md

```diff
@@ -82,33 +101,16 @@ Skills required to complete the project (you can apply without some of them):
 * Knowledge of [Git][] for storing your work on [GitHub][]
 * Knowledge of SurveyMonkey OAuth.
 
-Any of these skills could be learned during the project and I will help you
-with them, but you probably can't learn *all* of them during GSOC.
-
-### Mentors
-
-* [@pbanaszkiewicz](https://github.com/pbanaszkiewicz)
-  (timezone: UTC+1h [Poland])
-* [@rgaiasc](https://github.com/rgaiacs)
-  (timezone: UTC+0h [UK])
+Any of these skills could be learned during the project and we will help you
+with them, but you probably can't learn *all* of them during GSoC.
 
-### Interested in making this project? Here's what you should do
+### First steps
 
-If you want to show interest in making this project, we expect you to do
-a demo application. The application should grab responses from your
-SurveyMonkey survey and present them on the web.
-
-You'll need:
-
-1. a SurveyMonkey account, and one survey
-2. some test responses to the survey
-3. access tokens for your account
-4. public GitHub repository with your application
-
-Please make sure your account doesn't store any real information as you'll have
-to share the tokens publicly.
-
-Finally, please contact us and let us know the link to your repository.
+If you want to show interest in making this project,
+we expect you to have at lest a Django application somewhere on the internet.
+If the Django application grabs responses from your
+SurveyMonkey survey and present them on the web
+you will get a bonus.
```

Comment:
I agree with you here; your requirement is less strict and still provides us with input on how someone writes the code.

Some small typos fixed:
```markdown
If you want to show interest in making this project, we would like to see
your sample Django project.
As a bonus, you can implement grabbing responses from your SurveyMonkey
survey and presenting them.```

I'd still stick to the old title, though. I think it's more catchy.